Examples/Swift/ViewController.swift

```diff
@@ -45,220 +60,203 @@ class ViewController: UIViewController, MGLMapViewDelegate, NavigationViewContro
         
-        _ = Directions.shared.calculate(options) { [weak self] (waypoints, routes, error) in
+        _ = Directions.shared.calculate(options) { (waypoints, routes, error) in
             guard error == nil else {
                 print(error!)
                 return
             }
-            guard let route = routes?.first else {
-                return
-            }
-            
-            self?.userRoute = route
-            self?.startNavigationButton.isHidden = false
-            self?.simulateNavigationButton.isHidden = false
-            self?.howToBeginLabel.isHidden = true
+            guard let route = routes?.first else { return }
+                
+            self.currentRoute = route
             
             // Open method for adding and updating the route line
-            self?.mapView.showRoute(route)
-            
-            didFinish?()
+            self.mapView.showRoute(route)
         }
     }
     
-    func startNavigation(along route: Route, simulatesLocationUpdates: Bool = false) {
-        // Pass through:
-        // 1. The route the user will take.
-        // 2. A `Directions` object, used for rerouting.
+    // MARK: - Basic Navigation
+    
+    func startBasicNavigation() {
+        guard let route = self.currentRoute else { return }
+            
+        exampleMode = .default
+            
         let navigationViewController = NavigationViewController(for: route)
-        
-        // If you'd like to use AWS Polly, provide your IdentityPoolId below.
-        // `identityPoolId` is a required value for using AWS Polly voice instead of iOS's built in AVSpeechSynthesizer.
-        // You can get a token here: http://docs.aws.amazon.com/mobile/sdkforios/developerguide/cognito-auth-aws-identity-for-ios.html
-        //navigationViewController.voiceController?.identityPoolId = "<#Your AWS IdentityPoolId. Remove Argument if you do not want to use AWS Polly#>"
-        
-        navigationViewController.simulatesLocationUpdates = simulatesLocationUpdates
-        navigationViewController.routeController.snapsUserLocationAnnotationToRoute = true
-        navigationViewController.voiceController?.volume = 0.5
+        navigationViewController.simulatesLocationUpdates = simulationButton.isSelected
         navigationViewController.navigationDelegate = self
-        
-        // Uncomment to apply custom styles
-//        styleForRegular().apply()
-//        styleForCompact().apply()
-//        styleForiPad().apply()
-//        styleForCarPlay().apply()
-        
-        let camera = mapView.camera
-        camera.pitch = 45
-        camera.altitude = 600
-        if let userLocation = mapView.userLocation {
-            camera.centerCoordinate = userLocation.coordinate
-            if let location = userLocation.location {
-                camera.heading = location.course
-            }
-        }
-        navigationViewController.pendingCamera = camera
```

Comment:
`RouteMapViewController.viewWillAppear(_:)` falls back to calling `setDefaultCamera(_:)`, which only sets the altitude and pitch. It would be feasible for this method to also set the center coordinate to the route’s initial location and heading. However, it wouldn’t be feasible to set the camera to the user’s current location right off the bat, because the location manager determines the current location asynchronously.

Before this PR lands, we should modify `setDefaultCamera(_:)` to account for the route’s origin.